pkg/config/configuration.go

```diff
@@ -451,7 +451,7 @@ func getSpiffeID(ctx context.Context) (string, error) {
 
 // IsOperationAllowedByAccessControlPolicy determines if access control policies allow the operation on the target app
 func IsOperationAllowedByAccessControlPolicy(spiffeID *SpiffeID, srcAppID string, inputOperation string, httpVerb common.HTTPExtension_Verb, appProtocol string, accessControlList *AccessControlList) (bool, string) {
-	if accessControlList == nil {
```

Comment:
Please add a test for the new behavior so we don't regress in the future?

Reply:
along with unit-test, please do some manual e2e tests as well on standalone mode. 

1. http-http
2. grpc-grpc
3. http-grpc
4. grpc-http

Reply:
Verified these combinations. Added the UTs